bin/live/pycbc_live_supervise_single_significance_fits

```diff
@@ -0,0 +1,517 @@
+        check_trigger_files(trfits_files, test_options, controls)
+
+    if missed_files == 10:
+        # If more than 10 days between files, something wrong with analysis.
+        # warn and use fewer files - 10 here is chosen to be an unusual amount
+        # of time for the analysis to be down in standard operation
+        logging.warning('More than 10 days between files, only using '
+                        f'{found_files} files for combination!')
+
+    file_id_str = f'{first_date}-{end_date}'
+    if 'output-id-str' in controls:
+        file_id_str += f"-{controls['output-id-str']}"
+    out_fname = f'{file_id_str}-TRIGGER_FITS_COMBINED'
+    combined_options['output'] = os.path.join(output_dir, out_fname + '.hdf')
+
+    if not trfits_files:
+        raise ValueError("No files meet the criteria")
+
+    combined_options['trfits-files'] = ' '.join(trfits_files)
+
+    combined_args = ['pycbc_live_combine_single_significance_fits']
+    combined_args += dict_to_args(combined_options)
+
+    run_and_error(combined_args)
+
+    logging.info('Copying combined fits file to local filesystem')
+    try:
+        shutil.copyfile(
+            combined_options['output'],
+            controls['variable-trigger-fits']
+        )
+    except Exception as e:
+        mail_volunteers_error(
+            controls,
+            [str(e)],
+            "PyCBC live could not copy to variable trigger fits file"
+        )
+        raise e
+
+
+    logging.info(
+        "%s updated to link to %s",
+        controls['variable-trigger-fits'],
+        combined_options['output']
+    )
+
+    logging.info("Plotting combined fits")
+    # Add plotting for combined fits, and linking to the public directory
+    combined_plot_output = os.path.join(output_dir,
+                                        f"{{ifo}}-{out_fname}-{{type}}.png")
+    combined_plot_arguments = [
+        'pycbc_live_plot_combined_single_significance_fits',
+        '--combined-fits-file',
+        combined_options['output'],
+        '--output-plot-name-format',
+        combined_plot_output,
+        '--log-colormap'
+    ]
+
+    run_and_error(combined_plot_arguments)
+
+    combined_plot_outputs = [
+        combined_plot_output.format(ifo=ifo, type='fit_coeffs') for ifo in
+        combined_options['ifos'].split()
+    ]
+    combined_plot_outputs += [
+        combined_plot_output.format(ifo=ifo, type='counts') for ifo in
+        combined_options['ifos'].split()
+    ]
+
+    # Link the plots to the public-dir if wanted
+    if 'public-dir' in controls:
+        logging.info("Linking combined fits")
+        for cpo in combined_plot_outputs:
+            symlink(cpo, public_dir)
+
+    logging.info('Done')
+
+
+def wait_for_utc_time(target_str):
+    """Wait until the UTC time is as given by `target_str`, in HH:MM:SS format.
+    """
+    target_hour, target_minute, target_second = map(int, target_str.split(':'))
+    now = datetime.utcnow()
```

Comment:
I think utcnow is being deprecated actually so this may need now()

Reply:
For context, here is a nice [writeup](https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated). I would not worry about this just yet (this is going in production soon, and it will work for the rest of the run) but maybe adding a comment or opening an issue is warranted to plan ahead.